acceptance/extension-logging-fluentd/src/__tests__/fixtures/fluentd.docker.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/extension-logging
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import * as path from 'path';
+import {GenericContainer, StartedTestContainer} from 'testcontainers';
+
+export const ROOT_DIR = path.join(__dirname, '../../../fixtures');
+export const ETC_DIR = path.join(ROOT_DIR, 'etc');
+
+/* eslint-disable require-atomic-updates */
+async function startFluentd() {
+  if (process.env.FLUENTD_SERVICE_HOST != null) return;
+  const container = await new GenericContainer(
+    'fluent/fluentd',
+    'v1.7.4-debian-1.0',
+  )
+    .withName('fluentd_lb4')
+    .withExposedPorts(24224, 9880)
+    .withEnv('FLUENTD_CONF', 'fluentd.conf')
+    .withBindMount(ETC_DIR, '/fluentd/etc', 'ro')
+    .start();
+  process.env.FLUENTD_SERVICE_HOST = container.getContainerIpAddress();
+  process.env.FLUENTD_SERVICE_PORT_TCP = container
+    .getMappedPort(24224)
+    .toString();
+  process.env.FLUENTD_SERVICE_PORT_HTTP = container
+    .getMappedPort(9880)
+    .toString();
+  return container;
+}
+
+let fluentd: StartedTestContainer | undefined;
+
+/* eslint-disable no-invalid-this */
+/**
+ * Root-level before hook to start Fluentd container
+ */
+before(async function() {
```

Comment:
While this is better in the sense that we pay the price of Docker container/image startup only when explicitly running acceptance tests for fluentd, I find it still as a suboptimal developer experience.

Let's say there is a problem in the integration between our logging extension and the real Fluentd service that I want to troubleshoot. To do so, I may try different approaches:

1. Start Fluentd in a container, modify one of our example apps to use our logging extension, debug the app & extension to see what's going one.

2. Add additional logging, perhaps enable debug logs, and then run the acceptance tests in this directory. Tweak the code, re-run the tests to see if the problem is fixed.

In the first example, I don't have any easy & standard way how to start the container, because I cannot run the `before` hook outside of the test suite.

In the second example, I am waiting for Fluentd to start and stop every time I run the test suite. What a waste of my time (and CPU cycles of my computer).

To wrap it up: it's great that you moved Fluentd tests into `acceptance` folder 👍 Now let's extract the code starting (and stopping?) the docker container into a standalone script that we can run only once.